package.json

```diff
@@ -17,7 +17,8 @@
     "set-up": "npm run build",
     "server": "node server.js",
     "start": "npm run server",
-    "test": "jest --roots ./test/",
+    "//test": "echo 'NODE_OPTIONS is required until this issue is resolved: https://github.com/facebook/jest/issues/9430 '",
+    "test": "NODE_OPTIONS='--experimental-vm-modules' jest --roots ./test/",
```

Comment:
_non-blocking_

An ugly hack. :-) Just noting here that this env var will be needed when running individual test files too, e.g.:

    NODE_OPTIONS='--experimental-vm-modules' npx jest test/auspice_client_requests.test.js

Reply:
Good point, I updated the README in 781a0237afd1bbd6a069917c903a2ebd23864b90.